themes/default/content/docs/pulumi-cloud/webhooks.md

```diff
@@ -51,130 +45,85 @@ The Webhooks management page is on the Stack or Organization Settings tab.
+| `preview_failed`       | `stack_preview` | Both                       | When a stack `preview` fails.     |
+| `update_succeeded`     | `stack_update`  | Both                       | When a stack `update` succeeds.   |
+| `update_failed`        | `stack_update`  | Both                       | When a stack `update` fails.      |
+| `destroy_succeeded`    | `stack_update`  | Both                       | When a stack `destroy` succeeds.  |
+| `destroy_failed`       | `stack_update`  | Both                       | When a stack `destroy` fails.     |
+| `refresh_succeeded`    | `stack_update`  | Both                       | When a stack `refresh` succeeds.  |
+| `refresh failed`       | `stack_update`  | Both                       | When a stack `refresh` fails.     |
+| `deployment_queued`    | `deployment`    | Both                       | When a deployment is queued.      |
+| `deployment_started`   | `deployment`    | Both                       | When a deployment starts running. |
+| `deployment_succeeded` | `deployment`    | Both                       | When a deployment succeeds.       |
+| `deployment_failed`    | `deployment`    | Both                       | When a deployment fails.          |
 
-{{% /choosable %}}
-{{% choosable language python %}}
+## Webhook Formats
 
-```python
-import hashlib
-import hmac
-import base64
+When creating a webhook, you can choose between the generic JSON webhook payload or `slack`
+formatted events.
 
-message = bytes('...', 'utf-8')
-secret = bytes('...', 'utf-8')
+### Slack-formatted Webhooks
 
-hash = hmac.new(secret, message, hashlib.sha256)
-hash.hexdigest()
+Slack-formatted webhooks allow you to seamlessly integrate notifications about your Pulumi stacks and organizations
+into your Slack workspace by simply providing a [Slack incoming webhook URL](https://api.slack.com/messaging/webhooks)
+and optionally choosing which events you want delivered using [event filters](#event-filtering).
 
-expected_signature = base64.b64encode(hash.digest())
-```
+You can either create your own Slack app (or use an existing one you may already have installed in your workspace), or
+follow the link below to quickly get started with a pre-defined Slack app manifest.
 
-{{% /choosable %}}
-{{% choosable language go %}}
+[Slack App Manifest](https://api.slack.com/apps?new_app=1&manifest_yaml=display_information%3A%0A%20%20name%3A%20pulumi-slack-notifications%0A%20%20description%3A%20Funnel%20Pulumi%20webhooks%20to%20Slack%0A%20%20background_color%3A%20%22%238a3391%22%0Afeatures%3A%0A%20%20bot_user%3A%0A%20%20%20%20display_name%3A%20pulumi-slack-notifications%0A%20%20%20%20always_online%3A%20false%0Aoauth_config%3A%0A%20%20scopes%3A%0A%20%20%20%20bot%3A%0A%20%20%20%20%20%20-%20incoming-webhook%0Asettings%3A%0A%20%20org_deploy_enabled%3A%20false%0A%20%20socket_mode_enabled%3A%20false%0A%20%20token_rotation_enabled%3A%20false)
```

Comment:
@susanev I would like this link to be like a button or something. Any thoughts on how I might achieve this?

Reply:
@komalali im actively trying to remove html from our markdown. we can instead use a shortcode for buttons so we can keep things consistent.

for now though you should be able to grab similar html from the get started page with buttons.
https://github.com/pulumi/pulumi-hugo/blob/master/themes/default/content/docs/get-started/_index.md

pls use an o utline (secondary) button and not a filled-in (primary) button.
does that work for now?
@anita-trimbur and i can return to this in a few weeks and improve it!

Reply:
sounds good thanks!